universal-application-tool-0.0.1/conf/routes

```diff
@@ -139,11 +139,9 @@ POST    /callback/:client_name       controllers.CallbackController.callback(req
 
 # Log into application
 GET     /loginForm                   controllers.HomeController.loginForm(request: Request, message: java.util.Optional[String])
-GET     /idcsLogin                   controllers.LoginController.idcsLoginWithRedirect(request: Request, redirectTo: java.util.Optional[String])
```

Comment:
We need to be careful with removing existing routes, often times things like this need to be maintained for a release cycle as through the release different versions of the server may be available, as well if a route is effectively loaded into a users browser before a release, they may try to access it after.   That seems like a concern here correct?

This isn't something we have a good plan on afaik.

Reply:
Yes, that is definitely a concern. Should I leave the former routes in this file for now and then have a TODO to clean them up in the future?

Reply:
Yeah, if that works with your PR leaving them and cleaning up later is safest.

Reply:
So, looking at it now, I am realizing that in order to leave the routes, I'll also have to leave the deprecated code in LoginController. Is that something I should do? Wondering if we should discuss this specific code in our next daily sync.

Reply:
I would have to leave all the methods in LoginController that are referenced by the old routes for the code not to break. Sorry, not sure if my above comment made total sense.